src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/Features/IHttpRequestInputStreamFeature.cs

```diff
@@ -3,14 +3,19 @@
 
 #if NETCOREAPP
 
+namespace Microsoft.AspNetCore.SystemWebAdapters.Features;
+
+using System.Diagnostics.CodeAnalysis;
```

Comment:
using before namespace

Reply:
I'd noticed an inconsistency, but with the file scoped namespaces, I wasn't sure what the pattern was. I updated .editorconfig to enforce that